src/rdkafka_conf.c

```diff
@@ -284,7 +284,7 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
 	  _RK(broker_addr_ttl),
 	  "How long to cache the broker address resolving "
           "results (milliseconds).",
-	  0, 86400*1000, 1*1000 },
+	  0, 86400*1000, 60*1000 },
```

Comment:
DNS caching is performed by the resolver (typically libc) based on the records TTL setting, adding another layer of caching is inadvisable and will slow down the propagation of DNS record changes.

The initial default of broker.address.ttl was actually 5 minutes, but was decreased to 1s for this reason:
a0aff233e0be2f295e2e32e468ab22498c86c62c
https://github.com/edenhill/librdkafka/issues/494

The current default of 1s provides a bit of dampening, rather than caching.
A user that wants to dampen further can increase the configuration property.